core/runtime/v2/bridge.go

```diff
@@ -25,29 +25,29 @@ import (
 	"google.golang.org/protobuf/types/known/emptypb"
 
 	v2 "github.com/containerd/containerd/api/runtime/task/v2"
-	v3 "github.com/containerd/containerd/api/runtime/task/v3"
+	api "github.com/containerd/containerd/api/runtime/task/v3"
```

Comment:
Same for this one; also the first commit removed a `// Comment` that (from the looks of it) was informational, so probably should be kept.